docs/user-manual/en/SUMMARY.md

```diff
@@ -10,7 +10,7 @@
 * [Using the Server](using-server.md)
 * [Upgrading](upgrading.md)
 * [Address Model](address-model.md)
-* [Protocols and Interoperability](protocols-interoperability.md)
```

Comment:
but you did not rename the md here.. did you?

Reply:
a quick search, I see protocols-interoperability used in links in mqtt.md amqp.md etc

Reply:
I updated the PR, however I think these links show that the chapter *should not* be renamed.

ie "interoperability"  helps reader understand the scope of the doc, it's not just a list of supported protocols.

Why would i click on 'Supported protocols' to read about 'Configuration of max frame payload length for STOMP web-socket'?